packages/vue/src/components/basic/DownShift.jsx

```diff
@@ -358,11 +358,12 @@ export default {
 			Object.keys(stateToSet).map(key => {
 				// eslint-disable-next-line
 				this.isControlledProp(key)
-					? this.$emit(`${key}Change`, stateToSet[key])
+					? this.$emit(`${key}Change`, stateToSet[key]) && this.$emit(`${key}-change`, stateToSet[key])
```

Comment:
@royduin Can we remove `&&` operator and use `if else` condition to trigger events?